src/TTreeWidget.h

```diff
@@ -60,6 +61,20 @@ class TTreeWidget : public QTreeWidget
     void setIsKeyTree();
     void beginInsertRows(const QModelIndex& parent, int first, int last);
     void getAllChildren(QTreeWidgetItem*, QList<QTreeWidgetItem*>&);
+    int m_childID;
```

Comment:
Our existing style is a prefixed `m` only for member variables (with an "initial" capital letter for the first letter in the "name") - and we are trying now to initialise them in the header file if they do not need to be done from passed arguments in the constructor) e.g.:
```suggestion
    int mChildID = -1;
```